packages/jellyfish-api-core/__tests__/category/server/getRpcInfo.test.ts

```diff
@@ -0,0 +1,27 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import waitForExpect from 'wait-for-expect'
+
+describe('Server on masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getRpcInfo', async () => {
+    await waitForExpect(async () => {
```

Comment:
Are there any reasons for using `waitForExpect`? In the past, there were flaky conditions that require that, it's mostly resolved now. You shouldn't need to use this, it creates non-determinstic tests.

Reply:
> In the past, there were flaky conditions that require that, it's mostly resolved now.

Understood. I know that `waitForExpect` to meant to be used for asynchronous api calls or e2e tests therefore it seems appropriate to be included here. I suppose regular async await will suffice.

Reply:
For some context that might be relevant for you, the MasterNodeContainer automatically generate blocks to move the ledger forward periodically. We don't use `waitForExpect` as we want to deterministically reach a block at a height. 

E.g. send balance, use `container.generate(1)` to force a block generation to move balance. Then lastly check updated balances.